arcgis-ios-sdk-samples/Maps/Display device location with NMEA data sources/README.md

```diff
@@ -6,18 +6,22 @@ Parse NMEA sentences and use the results to show device location on the map.
 
 ## Use case
 
-NMEA sentences can be retrieved from GPS receivers and parsed into a series of coordinates with additional information. Devices without a built-in GPS receiver can retrieve NMEA sentences by using a separate GPS dongle, commonly connected bluetooth or through a serial port.
+NMEA sentences can be retrieved from GNSS/GPS receivers and parsed into a series of coordinates with additional information. Devices without a built-in GNSS receiver can retrieve NMEA sentences by using a separate GNSS dongle, commonly connected through Bluetooth or a serial port.
```

Comment:
Is this section supposed to be generic to any platform? It just seems odd that it references "a built-in GNSS receiver" and "a serial port" when iOS devices have neither of those things.